Cython/Utility/ImportExport.c

```diff
@@ -1,14 +1,3 @@
-/////////////// PyIdentifierFromString.proto ///////////////
```

Comment:
As far as I can tell, this is unused

Reply:
There used to be a `__Pyx_ImportModule()` that employed it. Probably still needed in 0.29.x.

Reply:
Looks like `__Pyx_PyIdentifier_FromString` is also unused in branch 0.29.x. 

```
$ git grep PyIdentifier
Cython/Compiler/ModuleNode.py:            h_code.put(UtilityCode.load_as_string("PyIdentifierFromString", "ImportExport.c")[0])
Cython/Utility/ImportExport.c://///////////// PyIdentifierFromString.proto ///////////////
Cython/Utility/ImportExport.c:#if !defined(__Pyx_PyIdentifier_FromString)
Cython/Utility/ImportExport.c:  #define __Pyx_PyIdentifier_FromString(s) PyString_FromString(s)
Cython/Utility/ImportExport.c:  #define __Pyx_PyIdentifier_FromString(s) PyUnicode_FromString(s)
```